index.html

```diff
@@ -1910,17 +1941,18 @@ <h3>WoT Binding Templates</h3>
 		</section>
 		<section>
 			<h3>WoT Scripting API</h3>
-			<p>The non-normative WoT Scripting API document proposes a
+			<p>The non-normative WoT Scripting API document defines a
 				programming interface that allows scripts to discover and consume
-				(retrieve) Thing Descriptions, instantiate local objects that act as
-				a proxy for remote Things and their Interaction Affordances
+				(retrieve) Thing Descriptions and instantiate local objects that act as
+				an interface to other Things and their Interaction Affordances
 				(Properties, Actions, and Events). It also allows scripts to specify
-				and implement Interaction Affordances and expose Things.</p>
+				and implement Interaction Affordances and expose Things (that is,
+                                define and publish a Thing Description).</p>
 			<p>
 				The <a>Scripting API</a> is an optional "convenience" building block
-				in WoT and it is typically used in network nodes that are able to
-				host a WoT Runtime, for instance gateways or more powerful IoT
-				devices.
+				in WoT.  It is used in combination with a WoT Runtime
```

Comment:
I wonder if we are making a constraint here. So far I understood that a WoT Runtime can exist without a scripting API implementation, just encapsulating the WoT network interface and the environment.